project-roles/trusted-committer.md

```diff
@@ -0,0 +1,162 @@
+# Title
+
+Trusted Comitter
+
+# Patlet
+
+Many inner-source projects will find themselves in a situation where
+they consistently receive feedback, features, and bug-fixes from a contributor.
+In these situations project maintainers seek ways to recognize and reward the
+work of the contributor above and beyond single contributions.
+
+# Context
+
+  - You are the maintainer of a cross-team library, service, or shared resource
+  - You receive regular contributions
+
+# Problem
+
+  - Project maintainers want to find ways to scale their ability to support a
+  project
+  - Project maintainers want to find ways to scale the longevity of the value
+  delivered by a project
+
+# Forces
+
+  - Over the lifecycle of a project the focus of the maintainers may shift away
+  to accomodate changing business priorities
+  - Contributors seek visible artifacts of their contributions demonstrating
+  value
+  - Motivated contributors looking to push further inner-sourced projects
+  - Maintaining a project of reasonable complexity is taxing for a small team
```

Comment:
I always thought small teams were the most productive ;). Is this point about scaling up in the face of increasing demand?

Reply:
Hah! Yes, this point addresses a small team delivering a project that experiences wide adoption throughout an organization.